DbInterface/__init__.py

```diff
@@ -0,0 +1 @@
+from .Neo4jStream import Neo4jStream
```

Comment:
Relative import (using . before the module name) could be problematic:
https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time

Reply:
used a module so its intended.

Reply:
done, btw you can suggest code edits if you'd like